src/gesture.js

```diff
@@ -173,6 +175,35 @@ export class Gestures {
     return overserver.add(handler);
   }
 
+  /**
+   * Unsubscribes all handlers from the given gesture recognizer. Returns
+   * true if anything was done. Returns false if there were no handlers
+   * registered on the given gesture recognizer in first place.
+   *
+   * @param {function(new:GestureRecognizer<DATA>, !Gestures)} recognizerConstr
+   * @return {boolean}
+   */
+  removeGesture(recognizerConstr) {
```

Comment:
Could this become a  `remove` (or `unlisten`) on `GestureRecognizer` class and hence on every recognizer instance?

My concern with current approach is that it gives one part of the code to unregister handlers for everyone which breaks code isolation.

Is there a use-case that unregistering handlers individually is not enough?

Reply:
chatted offline, `gesture` instance is tied to the `element` so this is fine.

Reply:
Anymore changes to address or are we good for merge? =)

Reply:
good to go, merged,